### PR TITLE
bug/EUI-7014

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7014

### Change description ###
Request reviewed by details not displaying when legal Ops/Admin SAR rejected/request more info

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
@Kartheek-S thanks for the changes

File: api/roleAccess/dtos/to-role-assignment-dto.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* tslint:disable:object-literal-sort-keys */
+import { UserInfo } from 'auth/interfaces/UserInfo';
```
Comment (connorpgpmcelroy) [inline api/roleAccess/dtos/to-role-assignment-dto.ts:2]: need relative folder '../auth' for example

**Review by DavidJayakumar — APPROVED** [review]

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Minor change to import needed

File: api/roleAccess/dtos/to-role-assignment-dto.ts (hunk 1 above)
Comment (Kartheek-S) [reply api/roleAccess/dtos/to-role-assignment-dto.ts]: Changes pushed

**Review by connorpgpmcelroy — APPROVED** [review]

**Pull request merged**